x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_multi_step_extension.tsx

```diff
@@ -6,13 +6,116 @@
  */
 
 import React, { memo } from 'react';
-import type { PackagePolicyCreateMultiStepExtensionComponentProps } from '@kbn/fleet-plugin/public';
+import {
+  EuiText,
+  EuiIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  useEuiTheme,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { css } from '@emotion/css';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../../../../common/lib/kibana';
+
+const CenteredEuiFlexItem = styled(EuiFlexItem)`
+  align-items: center;
+`;
+
+const LargeLogo = styled(EuiIcon)`
+  width: 128px;
+  height: 128px;
+`;
 
 /**
- * TBD: A component to be displayed in multi step onboarding process.
+ * A component to be displayed in multi step onboarding process.
  */
-export const EndpointPolicyCreateMultiStepExtension =
-  memo<PackagePolicyCreateMultiStepExtensionComponentProps>(({ newPolicy }) => {
-    return <></>;
-  });
+export const EndpointPolicyCreateMultiStepExtension = memo(() => {
+  const { docLinks } = useKibana().services;
+  const { size } = useEuiTheme().euiTheme;
+
+  const title = (
+    <EuiText>
+      <h3>
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.title"
+          defaultMessage="We'll save your integration with our recommended defaults."
+        />
+      </h3>
+    </EuiText>
+  );
+
+  const features = (
+    <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiIcon type="check" />
+      </EuiFlexItem>
+
+      <EuiFlexItem>
+        <EuiText size="m">
+          <p>
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.feature"
+              defaultMessage="Windows, Mac, and Linux event collection"
+            />
+          </p>
+        </EuiText>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  const details = (
+    <EuiText size="m" color="subdued">
+      <p>
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.multiStepOnboarding.details"
+          defaultMessage="You can change this later by editing the Endpoint Security integration agent policy.
+      Read more about Endpoint security configuration in our {docsPage}."
+          values={{
+            docsPage: (
+              <EuiLink href={docLinks.links.securitySolution.configureEndpointIntegrationPolicy}>
```

Comment:
Wondering if we should open the link in a new tab:

```typescript
<EuiLink href={docLinks.links.securitySolution.configureEndpointIntegrationPolicy} target="_blank" external> [...] </EuiLink>
```
cc: @caitlinbetz @kevinlog

Reply:
Yes, let's open on a new tab so that it doesn't take the user from the onboarding flow

Reply:
42ce7b0